components/gearalignmentdevice.py

```diff
@@ -30,6 +30,19 @@ def setup(self):
         """Run just after createObjects.
         Useful if you want to run something after just once after the
         robot code is started, that depends on injected variables"""
+        self.gear_alignment_motor.clearStickyFaults()
+
+        self.gear_alignment_motor.setControlMode(CANTalon.ControlMode.Position)
+        self.gear_alignment_motor.setFeedbackDevice(CANTalon.FeedbackDevice.AnalogPot)
+
+        self.gear_alignment_motor.setPID(20, 0, 0)
+
+        # self.gear_alignment_motor.enableLimitSwitch(True, True)
+        self.gear_alignment_motor.enableLimitSwitch(False, False)
+
+        self.setpoint = self.gear_alignment_motor.getPosition()
+
+        self.gear_alignment_motor.reverseSensor(True)
```

Comment:
This code seems to be duplicated in `on_enable`. I'm guessing you probably want to remove some of the code in `on_enable` as well.